Core/BattlegroundIC: Removed 1 hack:

Golrag opened the pull request:
- Parachute spell is now blizzlike
- Also restored "The battle will begin in 2 minutes."

The positions I used are made up, not from sniffs !

Golrag committed:
Core/BattlegroundIC: Removed 1 hack:
* Parachute spell is now blizzlike
* Also restored "The battle will begin in 2 minutes."

Kittnz commented:
Should be ported to 3.3.5 too

Nyeriah commented:
Does these areatrigger changes belong to this PR?

Golrag commented:
While I was in IoC to test this, I found out that the message to announce that the battle will begin in 2 minutes was gone and changed to "You have left areatrigger %u". This was only in IoC so I decided to put it in this PR.

Nyeriah commented:
Did you have debug areatrigger on by any chance? Well, can't confirm that at 3.3.5

Golrag commented:
The areatrigger debug output for entering en leaving is 6.x, see https://github.com/TrinityCore/TrinityCore/commit/41e2b0046dc73ab2adf9c15442d70a7b769013f9

Trinity_string 1205 was already used and I missed that part in the PR for the AreaTriggers. So I changed it back and I changed 1204 to 'support' both leaving and entering areatrigger (so it won't break it :)).

I can move it to another PR if you really want it, but it only affected IoC, that's why I put it in this one :)

Shauren commented on src/server/scripts/Northrend/isle_of_conquest.cpp:
```diff
@@ -152,28 +153,11 @@ class spell_ioc_gunship_portal : public SpellScriptLoader
+                if (GetHitCreature()->GetTransport())
+                    GetHitCreature()->GetTransport()->AddPassenger(caster);
+
+                caster->TeleportTo(GetHitCreature()->GetMapId(), GetHitCreature()->GetTransOffsetX(), GetHitCreature()->GetTransOffsetY(), GetHitCreature()->GetTransOffsetZ(), GetHitCreature()->GetTransOffsetO(), TELE_TO_NOT_LEAVE_TRANSPORT);
```
How nice of you to teleport players to the middle of nowhere. Player::TeleportTo does NOT support taking transport offsets as arguements, it must always be absolute position

Golrag replied:
Just tested it again, and got teleported to the gunship ^^, should I change it and make it like the player only gets teleported if there is a transport ?

Edit: found out that when the battleground ends I get teleported to the middle of nowhere

Shauren replied:
This is because client takes the packet nicely but serverside position is in middle of nowhere

Golrag replied:
Oh.. revert this change then ? :)

joschiwald commented on src/server/game/Handlers/MiscHandler.cpp:
```diff
@@ -531,7 +531,7 @@ void WorldSession::HandleAreaTriggerOpcode(WorldPackets::Misc::AreaTrigger& pack
 
     if (player->isDebugAreaTriggers)
-        ChatHandler(player->GetSession()).PSendSysMessage(packet.Entered ? LANG_DEBUG_AREATRIGGER_ENTERED : LANG_DEBUG_AREATRIGGER_LEFT, packet.AreaTriggerID);
+        ChatHandler(player->GetSession()).PSendSysMessage(LANG_DEBUG_AREATRIGGER_TRIGGERED, packet.Entered ? "entered" : "left", packet.AreaTriggerID);
```
use 2 trinity_strings to allow localization of the "entered" and "left" string

Golrag replied:
Should I use two consecutive trinity_string entries or not ?

Golrag replied:
Done ! :)
